src/piper.js

```diff
@@ -15,4 +15,21 @@ async function run () {
   }
 }
 
+function getDownloadUrl() {
+  const version = core.getInput('version')
+  if (version === 'release') {
+    console.log("Downloading latest release of piper")
+    return 'https://github.com/SAP/jenkins-library/releases/latest/download/piper'
+  } else if (version === 'master') {
+    console.log("Downloading latest build of master branch of piper")
+    return 'https://github.com/SAP/jenkins-library/releases/latest/download/piper_master'
```

Comment:
Extract release link `https://github.com/SAP/jenkins-library/releases/` to `const`.